src/app/sitemap.ts

```diff
@@ -0,0 +1,111 @@
+import type { MetadataRoute } from "next";
+import type { CategoryType } from "@/lib/types/categories";
+
+export default async function sitemap(): Promise<MetadataRoute.Sitemap> {
+  const apiURL = process.env.GRAPHQL_API_URL || "";
+  const baseURL = "https://volt-eosin.vercel.app";
+  const latestDate = new Date("2024-11-30T00:00:00.000Z");
```

Comment:
Posts have `modified` attribute, use that
![image](https://github.com/user-attachments/assets/ed7c9564-9a07-439d-9912-55af98419e05)
Check if other data also have this attribute, if not, don't add lastmod field.